scrapers/brickyates.yml

```
@@ -0,0 +1,33 @@
+name: brickyates
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - brickyates.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class="videoDetails clear"]/h3
+     Date:
+        selector: //div[@class='videoInfo clear']/p[1]
+        postProcess:
+          - replace:
+              - regex: "Date Added:"
+                with: ""
+          - parseDate: January 2, 2006
+     Performers:
+      Name: //li[@class="update_models"]/a
+     Details: //div[@class="videoDetails clear"]/p
+     Tags:
+      Name: //div[@class="featuring clear"][2]//a
```

```suggestion
      Name: //ul[contains(li,"Tags:")]//a
```
is probably safer to use